elementary/monitor/alerts/report_link_utils.py

```diff
@@ -1,37 +1,48 @@
+from enum import Enum
 from typing import Optional
 
 from elementary.monitor.alerts.schema.alert import ReportLinkData
 
-LINK_TEXT = "View test runs"
+TEST_LINK_TEXT = "View test runs"
+MODEL_LINK_TEXT = "View model runs"
+
+
+class ReportPath(Enum):
+    TEST = "test-runs"
```

Comment:
I think those should have their full names (`TEST_RUNS`, `MODEL_RUNS`) so that it wouldn't confuse with the test results screen.